src/main/java/com/buinak/imageapi/controller/ImageController.java

```diff
@@ -29,6 +29,12 @@ public ImageController(ImageService imageService) {
         return ResponseEntity.ok().body(imageService.addImage(name, description, file));
     }
 
+    @GetMapping(path = "getImageLinks")
```

Comment:
endpoints should be all in lower case, can you refactor all the endpoints? https://nordicapis.com/10-best-practices-for-naming-api-endpoints/